.github/workflows/releaseutils.yml

```diff
@@ -0,0 +1,30 @@
+name: Add sbom
+
+on:
+  workflow_dispatch:
+  workflow_run:
+        workflows: [test-latest-release]
+        types: [completed]
+
+permissions: read-all
+
+jobs:
+    add-sbom:
+        name: Add SBOM
+        runs-on: ubuntu-latest
+        if: ${{github.event.workflow_run.conclusion=='success'}}
+        steps:
+            - name: checkout code
+              uses: actions/checkout@v4
+
+            - name: Create SBOM
+              uses: advanced-security/sbom-generator-action@v0.0.1
+              id: sbom
+              env: 
+                  GITHUB_TOKEN: ${{ github.token }}

```

Comment:
i think with read-all permissions for the token are said to 'read', since we only need to read the token while doing this it won't thwart that

Reply:
As for the github token, I provided since its mention that way in its documentation, I can look for a different github action for the same , if you want me to.